strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/AbstractArtifactManagementService.java

```diff
@@ -419,4 +420,32 @@ public void delete(String storageId,
         }
     }
 
+    @Override
+    public RepositoryFileAttributes getAttributes(String storageId,
+                               String repositoryId,
+                               String path)
+            throws ArtifactTransportException,
+                   ProviderImplementationException, NoSuchAlgorithmException
+    {
+        try
+        {
+            try
+            {
+                return artifactResolutionService.getAttributes(storageId, repositoryId, path);
+            }
+            catch (IOException e)
+            {
+                // This is not necessarily an error. It could simply be a check
+                // whether a resource exists, before uploading/updating it.
+                logger.debug("The REQUESTED path does not exist: - /" + storageId + "/" + repositoryId + "/" + path);
```

Comment:
`logger.debug("The requested path does not exist: /" + storageId + "/" + repositoryId + "/" + path);`